language/stdlib/modules/libra_system.mvir

```diff
@@ -24,7 +24,18 @@ module LibraSystem {
 
     resource BlockMetadata {
       // Height of the current block
+      // TODO: Should we keep the height?
```

Comment:
I think this is useful for a couple of reasons:
- users need a coarser notion of time; e.g., just something that's monotonically increasing
- we can specify and enforce policies like a "minimum epoch duration" using blocks

Reply:
It seems to me that we can just keep this construct in Move given Sam's suggestion. It doesn't need to come from the consensus tho.